guidelines/_sources/05_use-human-validation-for-llm-outputs.tex

```diff
@@ -6,57 +6,79 @@ \subsection{Use Human Validation for LLM Outputs}
-However, researchers should use systematic approaches to decide what and how human annotations can be replaced with LLM-generated ones, such as the methods proposed by Ahmed et al.~\cite{DBLP:journals/corr/abs-2408-05534}.
-
-When evaluating the capability of LLMs to generate SE-related artifacts, may employ human validation to complement machine-generated measures.
-For example, proxies for software quality, such as code complexity or the number of code smells, may be complemented by human ratings of maintainability, readability, or understandability.
-In the case of more abstract variables or psychometric measurements, human validation may be the only way of measuring a specific construct.
-For example, measuring human factors such as trust, cognitive load, and comprehension levels may inherently require human evaluation.
-
-When conducting empirical measurements, researchers should clearly define the construct that they are measuring and specify the methods used for measurement. 
-Further, they should use established measurement methods and instruments that are empirically validated~\cite{DBLP:journals/fcomp/HoffmanMKL23, DBLP:conf/chi/PerrigSB23}.
-Measuring a construct may require aggregating input from multiple subjects. 
-For example, a study may assess inter-rater agreement using measures such as Cohen's Kappa or Krippendorff’s Alpha before aggregating ratings.
-In some cases, researchers may also combine multiple measures into single composite measures.
-As an example, they may evaluate both the time taken and accuracy when completing a task and aggregate them into a composite measure for the participants' overall performance.
-In these cases, researchers should clearly describe their method of aggregation and document their reasoning for doing so.
-
-When employing human validation, additional confounding factors should be controlled for, such as the level of expertise or experience with LLM-based applications or their general attitude towards AI-based tools.
+
+Researchers \should employ human validation of LLM outputs whenever there is no established reference dataset or when automated metrics alone cannot fully capture the target qualities they are concerned with.
+These cases may include user-facing interactive studies (e.g., debugging or collaborative coding agents), auditing bias or fairness, and controlled experiments on developer productivity or quality (e.g., usefulness evaluation of LLM-based IDE plugin).
+However, researchers \should use systematic approaches to decide how human annotations can be replaced with LLM-generated ones, such as the methods proposed by Ahmed et al.~\cite{DBLP:journals/corr/abs-2408-05534}.
+Their suggested method involves using a jury of three LLMs with 3-4 few-shot examples rated by humans, where the model-to-model agreement on all samples is determined using Krippendorff's Alpha.
+Based on the level of agreement between the LLMs, more or less human ratings are replaced with LLM outputs.
```

Comment:
the line "where more or less human ratings are replaced" doesn't read well. DO you mean "human ratings can be replaced" or maybe "human ratings are replaceable". I think it would be good to commit to a recommendation - should we replace or not

Reply:
I've made it a bit more clear, if the agreement is high, we can replace them, if not, then we don't.